File: api/java/digital-oneid/src/main/java/digital/oneid/model/CompanyCreateRequest.java
package digital.oneid.model;

/**
 * Created by hubinotech on 01/05/20.
 */
public class CompanyCreateRequest {
    private String loginname;
    private String password;
    private String companyName;
    private String email;
    private String address;

    public String getLoginname() {
        return loginname;
    }
    public void setLoginname(String loginname) {
        this.loginname = loginname;
    }

    public String getPassword() {
        return password;
    }
    public void setPassword(String password) {
        this.password = password;
    }

    public String getCompanyName() {
        return companyName;
    }
    public void setCompanyName(String companyName) {
        this.companyName = companyName;
    }

    public String getEmail() {
        return email;
    }
    public void setEmail(String email) {
        this.email = email;
    }

    public String getAddress() {
        return address;
    }
    public void setAddress(String address) {
        this.address = address;
    }
}

File: APIDemo/BankStatementsAPIDemo/BankTransactionManagement/models/CompanyCreateRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BankTransactionManagement.models
{
    public class CompanyCreateRequest : BaseClass
    {
        public String loginname { get; set; }
        public String password { get; set; }
        public String companyName { get; set; }
        public String email { get; set; }
        public String address { get; set; }
    }
}
